### ISSUE TITLE
pvc_class patch

### ISSUE THREAD

**Issue by stefanodalli** [a3rev/page-views-count#10]
Hi there,
Due to a little bug in showing the correct today counter, I made the following patch inside
public static function pvc_get_stats( $post_id )
Take a look at:
```
============================================
...
if ( ! empty( $today ) ) {
$output_html .= number_format($today) . ‘ ‘ .__(‘views today’, ‘page-views-count’);
...
============================================
```
See attached file pvc_class.zip (contains pvc_class.php)
[pvc_class.zip](https://github.com/a3rev/page-views-count/files/1725481/pvc_class.zip)

**Comment by a3rev** [comment]
@stefanodalli 

Hi - just put out new version 2.0.4 with your patch - have acknowledged you in the change log for contributing the patch.

**Comment by stefanodalli** [comment]
OK. Thanks. Well done.
Bye
Stefano
^^^^^^^^^^^^^^^^^^^^^^^^

**Comment by a3rev** [comment]
Released in [version 2.0.4](https://github.com/a3rev/page-views-count/releases/tag/2.0.4)
